docs/index.md

```diff
@@ -16,115 +22,237 @@
+This is different from other embeddings, because PCA, MDS, and t-SNE do not set a threshold for genetic diversity, and instead scale the data based on the largest genetic distance disparity (this makes UMAP’s embedding so densely clustered).
+
+
+## Figure One
+
+<iframe src="https://blab.github.io/cartography/FullLinkedChartClickableFlu.html" style="width: 1200px; height: 400px;" frameBorder="0"></iframe>
+
+## Figure Two
+
+![](FullScatterplotFlu.png)
+
+## Figure Three
+![](FullViolinPlotFlu.png)
+
+
+
+## Zika:
+
+Zika: Zika in this project is used as a test case. 
+As zika genes all exist on the same genome and zika’s genome is considerably longer than H3N2 Influenza genome, the genetic diversity of zika, while low enough to create phylogenetic transmission tree, clades are hard to distinguish from each other - zika transmission trees are often colored by region and country for this reason. 
+It can be reasonably assumed that zika is a good test case for Cartography.
+
+## SUMMARY OF RESULTS FOR ZIKA: 
+
+PCA: The data was normalized and scaled, and PCA was run on it (Figure 4). 
+The PCA pairwise vs euclidean distance plot gave a pearson coefficient of .005, which was the lowest coefficient of the 4 embeddings analyzed. 
+The data points did not have any correlation to each other, meaning the data showed no signs of clustering at all (Figure 5). 
+The ratio between the medians given by the within vs between violin plot for PCA was 1:1, meaning there is no distinction between between clade and within clade distance, corroborating previous conclusions (Figure 6).
+
+MDS: MDS reduced the pairwise distance matrix (Figure 4). 
+The MDS pairwise vs euclidean distance plot gave a pearson coefficient of .878, which was the highest of the 4 embeddings. 
+The data points followed the LOESS line in an extremely linear fashion, which, given that MDS favors retaining global patterns over local patterns and therefore genetic distances should not differ widely from euclidean distance in the plot, upholds preexisting beliefs about the algorithm (Figure 5). 
+The ratio between the medians given by the within vs between violin plot for MDS was 7:1, revealing very tightly clustered data. 
+Some data points from different clades did overlap each other, but given that MDS retains global structure, this result is expected.
+The density of the within clade violin plot is at lower euclidean distances than the density of the between clade violin plot, revealing that it is quite easy to distinguish the between vs within clade measure given a euclidean distance (Figure 6). 
+MDS translated genetic distance to euclidean distance exactly - this pattern is clearly shown in the pairwise vs euclidean graph for MDS, as the LOESS line and the data points surrounding it have a linear 1:1 relationship between genetic and euclidean distance.
```

Comment:
Be careful about statements like this where you haven't actually performed a test to compare two distributions. In this case, the genetic and Euclidean distances are not _exactly_ the same or you could get a correlation of 1.0 and all the points would fall on the line of y = x. It is enough to state that these two variables are strongly correlated and provide the values that support your statement.